### PR TITLE
updated boardgames to board-games in a few places

### PR DESCRIPTION
**Pull request by jarrodkohl**
-boardGames index
-boardgame tile
-root router

boardgames to board-games

### CODE REVIEW

File: client/src/components/BoardGameTile.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const BoardGameTile = props => {
     const { id, name, rating } = props
     return(
         <div>
-            <li><Link to={`/boardgames/${id}`}>{name}</Link></li>
+            <li><Link to={`/board-games/${id}`}>{name}</Link></li>
```
Comment (dovakeen118) [inline client/src/components/BoardGameTile.js:8]: doesn't look like the Express clientRouter knows about this show route for board games (it has just "/board-games"

**Review by dovakeen118 — COMMENTED** [review]
confirm all routes match!

File: client/src/components/BoardGameShowPage.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                <li>Category {boardGame.category}</li>
+                <li>Players: {boardGame.minPlayers}-{boardGame.maxPlayers}</li>
+                <li>Est Play Time: {boardGame.estimatedPlayTime}</li>
+                <li>Rating: </li>
+            </ul>
+            <div className="description">
+            <p>Description:</p>
+            <p>{boardGame.description}</p>
+            </div>
+        </div>
```
Comment (nwalberts) [inline client/src/components/BoardGameShowPage.js:42]: this div is not doing anything, replace with a React fragment unless you are using it for styling

File: client/src/components/BoardGameShowPage.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    )
+
```
Comment (nwalberts) [inline client/src/components/BoardGameShowPage.js:44]: remove this extra space

File: client/src/components/BoardGameShowPage.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    useEffect(() =>{
+        getBoardGame()
+    }, [])
+
+    return(
+        <div>
+            <h2>{boardGame.name}</h2>
+            <ul className="game-info">
+                IMG placeholder
```
Comment (nwalberts) [inline client/src/components/BoardGameShowPage.js:32]: no placeholders. remove until you have this feature working

File: client/src/components/BoardGameShowPage.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+                <li>Category {boardGame.category}</li>
+                <li>Players: {boardGame.minPlayers}-{boardGame.maxPlayers}</li>
+                <li>Est Play Time: {boardGame.estimatedPlayTime}</li>
+                <li>Rating: </li>
```
Comment (nwalberts) [inline client/src/components/BoardGameShowPage.js:36]: no place holders , remove until you have this feature working

File: client/src/components/BoardGameShowPage.js (hunk 2 above)
Comment (jarrodkohl) [reply client/src/components/BoardGameShowPage.js:42]: We made it for styling, though we have not actually styled yet.

**Pull request merged**